scripts/merge-measures-data.js

```diff
@@ -47,9 +47,70 @@ function mergeQpp(qppJson) {
       if (performanceRateInfo.metricType) {
         qppJson[index].metricType = performanceRateInfo.metricType;
       }
+
     }
   });
 
   console.error('did not find measure details for the following', measuresNotFound);
+
+  // separate passes for ecqm data to keep things simple
+  // load extracted ecqm data
+  const generatedEcqmStrataJson = JSON.parse(fs.readFileSync(path.join(__dirname, '../util/generated-ecqm-data.json'), 'utf8'));
```

Comment:
Does it make sense to handle this inline with the other performance rate data file processing (line 30)? Right now the `measuresNotFound` error is compiling measures that are likely represented here.